### PR TITLE
GTEST/UCP/SOCKADDR: Increase flush timeout

### PR DESCRIPTION
**Pull request by Artemy-Mellanox**

### CODE REVIEW

File: test/gtest/ucp/test_ucp_sockaddr.cc [hunk 1]
```diff
@@ -480,7 +480,7 @@ class test_ucp_sockaddr : public ucp_test {
 
     void one_sided_disconnect(entity &e, enum ucp_ep_close_mode mode) {
         void *req           = e.disconnect_nb(0, 0, mode);
-        ucs_time_t deadline = ucs_time_from_sec(10.0) + ucs_get_time();
+        ucs_time_t deadline = ucs_time_from_sec(15.0) + ucs_get_time();
```
Comment (yosefe) [inline test/gtest/ucp/test_ucp_sockaddr.cc:483]: is this causing test failures?
maybe 60 sec?

**Comment by yosefe** [comment]
bot:pipe:retest

File: test/gtest/ucp/test_ucp_sockaddr.cc (hunk 1 above)
Comment (yosefe) [reply test/gtest/ucp/test_ucp_sockaddr.cc:483]: @Artemy-Mellanox WDYT?

**Comment by Artemy-Mellanox** [comment]
bot:retest

**Review by yosefe — APPROVED** [review]

**Pull request merged**